From: Rob Evers <revers@redhat.com>
Date: Wed, 24 Nov 2010 21:24:08 -0500
Subject: [scsi] lpfc: Update version to 8.2.0.87.1p
Message-id: <1290633848-10834-8-git-send-email-revers@redhat.com>
Patchwork-id: 29620
O-Subject: [RHEL5.6 PATCH 7/7] lpfc: Update lpfc version to 8.2.0.87.1p
Bugzilla: 655119

From: Rob Evers on behalf of Emulex <revers@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=655119

diff --git a/drivers/scsi/lpfc/lpfc_version.h b/drivers/scsi/lpfc/lpfc_version.h
index a981412..36d6c71 100644
--- a/drivers/scsi/lpfc/lpfc_version.h
+++ b/drivers/scsi/lpfc/lpfc_version.h
@@ -18,7 +18,7 @@
  * included with this package.                                     *
  *******************************************************************/
 
-#define LPFC_DRIVER_VERSION "8.2.0.87"
+#define LPFC_DRIVER_VERSION "8.2.0.87.1p"
 
 #define LPFC_DRIVER_NAME		"lpfc"
 #define LPFC_SP_DRIVER_HANDLER_NAME	"lpfc:sp"
